minorminer/utils/raster_embedding.py

```diff
@@ -16,9 +16,97 @@
 import dwave_networkx as dnx
 import numpy as np
 import warnings
+import matplotlib.pyplot as plt
 from minorminer.subgraph import find_subgraph
 
+def visualize_embeddings(H, topology=None, title=None):
+    """
+    Visualizes the embeddings produced using dwave_networkx's layout 
+    according to the specified topology.
+
+    Parameters:
+    -----------
+    H : networkx.Graph
+        Input graph to be visualized on the topology.
+    topology : str, optional
+        The type of topology for visualization. 
+    title : str, optional
+        Title of the plot. Defaults to None.
+
+    """
+
+    if topology == 'chimera':
+        graph_size=(1, 1, 4)
+        # Set up the figure and axis
+        fig, ax = plt.subplots(figsize=(8, 6))
+        
+        # Create the Chimera topology graph (adjust size as needed)
+        G = dnx.chimera_graph(*graph_size)  
+        
+        # Draw the base Chimera unit cell
+        dnx.draw_chimera(G, node_color='r', ax=ax, label='Unit Cell')
+
+        # Draw the input graph on top of the unit cell
+        dnx.draw_chimera(
```

Comment:
do we require different parameters per topology?
if not, u can pull out the draw functions like
```
if topology == "chimera":
    draw = dnx.draw_chimera
elif topology == "zephyr":
   draw = dnx.draw_zephyr
...
draw(G, other_params=...)
```